_includes/configuration/toolbar-location.md

```diff
@@ -4,11 +4,13 @@
 
 The `toolbar_location` option is used to position the [toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar) and [menubar]({{site.baseurl}}/configure/editor-appearance/#menubar). The default behavior is to position the toolbar and menu bar above the content area. Setting this option to `bottom` positions the toolbar and menu bar below the content area. This option works in [classic editor]({{site.baseurl}}/general-configuration-guide/use-tinymce-classic/) (Iframe) mode and inline mode.
 
+When using `auto`, the menu bar and toolbar will be positioned above the content area by default. However, if there will never be enough space above to render the components, the menu bar and toolbar will be positioned at the the bottom instead.
```

Comment:
This _might_ be a better way to phrase this?

>Setting this option to `auto` positions the toolbar and menu bar either:
>- Above the content area (default)
>- Below the content area if the editor does not have enough space within the current frame to render the components above the editor.

Reply:
Fixed :+1: